sensor/common/scan/scan.go

```diff
@@ -47,6 +49,19 @@ var (
 	ErrEnrichNotStarted = errors.New("enrich was not started")
 
 	log = logging.LoggerForModule()
+
+	// sensorCentralScanMetricsLabel is used when collecting metrics about the semaphore for image scans
+	// that are triggered by sending a request to scanner over central.
+	sensorCentralScanMetricsLabel = prometheus.Labels{
+		"subsystem": "sensor",
+		"entity":    "image-scan-over-central",
+	}
+	// sensorAdHocScanMetricsLabel is used when collecting metrics about the semaphore for delegated image scans
+	// that are triggered by sending a request to the scanner instance running in the secured cluster.
+	sensorAdHocScanMetricsLabel = prometheus.Labels{
+		"subsystem": "sensor",
+		"entity":    "delegated-image-scan",
+	}
```

Comment:
Let's clean up the verbiage here - everything in `scan.go` is for handling a 'delegated scan'. If Sensor is handling the scan, it is a 'delegated scan'.

The distinction here is for what is requesting the scan, was it Central (from adhoc requests via roxctl, api, etc.) or Sensor (from observed k8s workloads, reprocessing, etc.). 

scanSemaphore = Delegated scans requested by Sensor
adHocScanSemaphore = Delegated scans requested by Central

Reply:
Thanks for that clarification. I added one more label to track the origin of the request in [8b93928](https://github.com/stackrox/stackrox/pull/14695/commits/8b93928c3a492ae09ef77a64488daa9438d8376c)